www/source/partials/docs/_commands-cli.html.md.erb

```diff
@@ -721,10 +721,255 @@ Exports the package to the specified format
 
 **ARGS**
 
-    <FORMAT>       The export format (ex: aci, cf, docker, kubernetes, mesos, or tar)
+    <FORMAT>       The export format (ex: aci, cf, docker, helm, kubernetes, mesos, or tar)
     <PKG_IDENT>    A package identifier (ex: core/redis, core/busybox-static/1.42.2) or filepath
                     to a Habitat Artifact (ex: /home/acme-redis-3.0.7-21120102031201-x86_64-linux.hart)
 
+**Read More**
+
+- [docker](#hab-pkg-export-docker) Creates a Docker image from a set of Habitat packages
+- [kubernetes](#hab-pkg-export-k8s) Creates a Docker image and Kubernetes manifest for a Habitat package
+- [helm](#hab-pkg-export-helm) Creates a Docker image and generates a Helm chart for the specified Habitat package
+
+<h2 id="hab-pkg-export-docker" class="anchor">hab pkg export docker</h2>
+Creates (an optionally pushes) a Docker image from a set of Habitat packages
```

Comment:
Really was not aware you can export multiple Habitat pkgs?

Reply:
with docker exporter, you can.